app/views/rating_questions/index.html.erb

```diff
@@ -0,0 +1,18 @@
+<div class="form">
+  <div class="flash-notice"><%= flash[:notice] %>
+  </div>
+  <h4 class="new_question"><%= link_to "New Question", new_rating_question_path %></h4>
+  <div class='questions'>
+    <% @rating_questions.each do |question| %>
+      <div class="single">
+          <h4><%= question.title %></h4>
+          <h4><%= question.tag %></h4>
+          <h4><%= link_to "Show", rating_question_path(question) %></h4>
```

Comment:
Consider using the shorter versions of the routing helpers here:

```
<%= link_to "Show", question %>
<%= link_to "Edit", [:edit, question] %>
<%= link-to "Delete", question %>
```

Reply:
Ive seen this syntax. I think I felt that it wasn't very readable. Is this the preferred syntax because its shorter?